shim.php

```diff
@@ -10,6 +10,44 @@ function toXPath($cssExpr, $prefix = 'descendant-or-self::') {
     }
 }
 
+// Add aliases for PHPUnit 6
+
+namespace {
+    if (version_compare(PHP_VERSION, '7.0.0') >= 0 && class_exists('PHPUnit\Framework\TestCase')) {
```

Comment:
I think that `if (!class_exists('PHPUnit_Framework_TestCase') && class_exists('PHPUnit\Framework\TestCase'))` condition would be more accurate.